### DISCUSSION TITLE
epoptes fails

### DISCUSSION THREAD

**Discussion by brcisna** [ltsp/ltsp#395] [General]
Hello All,

Debian Bullseye 

This is actually an epoptes problem,looked in epoptes forums but not sure what the actual fix is?
Started about 2-3 days ago,no clients are detected by Epoptes  .

Error:  Epoptes-client connecting to server:789... socat[3598] E Failed to set SNI host ""

Thank You

**Comment by vagrantc** [comment]
On 2021-02-03, brcisna wrote:
> This is actually an epoptes problem,looked in epoptes forums but not
> sure what the actual fix is?
>
> Started about 2-3 days ago,no clients are detected by Epoptes  .
>
> Error:  Epoptes-client connecting to server:789... socat[3598] E Failed to set SNI host ""

This was due to a socat version update, and is fixed in epoptes 21.02:

  https://github.com/epoptes/epoptes/issues/127

  https://bugs.debian.org/981612

Currently only in Debian unstable, but should migrate in a few days.


live well,
  vagrant

**Comment by alkisg** [comment]
Thanks to both, converting to a discussion as it's not related to LTSP.
